src/panels/config/helpers/forms/ha-input_select-form.ts

```diff
@@ -66,18 +73,104 @@ class HaInputSelectForm extends LitElement {
             "ui.dialogs.helper_settings.generic.icon"
           )}
         ></ha-icon-input>
-        <paper-input
-          .value=${this._initial}
-          .configValue=${"initial"}
-          @value-changed=${this._valueChanged}
-          .label=${this.hass!.localize(
-            "ui.dialogs.helper_settings.generic.initial_value"
-          )}
-        ></paper-input>
+        ${this.hass!.localize(
+          "ui.dialogs.helper_settings.input_select.options"
+        )}:
+        ${this._options.length
+          ? this._options.map((option, index) => {
+              return html`
+                <paper-item>
+                  <paper-item-body> ${option} </paper-item-body>
+                  <paper-icon-button
+                    .index=${index}
+                    .title=${this.hass.localize(
+                      "ui.dialogs.helper_settings.input_select.remove_option"
+                    )}
+                    @click=${this._removeOption}
+                    icon="hass:delete"
+                  ></paper-icon-button>
+                </paper-item>
+              `;
+            })
+          : html`
+              <paper-item>
+                ${this.hass!.localize(
+                  "ui.dialogs.helper_settings.input_select.no_options"
+                )}
+              </paper-item>
+            `}
+        <div class="layout horizontal bottom">
+          <paper-input
```

Comment:
What happens when I'm in this form and press enter, does it trigger an add of this item?